### ISSUE TITLE
Support for 4.14 kernel

### ISSUE THREAD

**Issue by devlithium** [imageguy/fglrx-for-Fedora#16]
Since i'm in 4.14 version i cannot complete the steps, will you support this version?

**Comment by imageguy** [comment]
The patch and install should work on 4.14, since the 4.13 patch includes all the changes we have. Give it a try and let me know if anything breaks.

**Comment by mirh** [comment]
Ehrm, are you sure you aren't confusing between 4.14 and 4.15 (which didn't need anything more)
Because afaik on arch there was [this](https://aur.archlinux.org/cgit/aur.git/tree/4.14-npfeiler-task_struct-mm_segment_t.patch?h=catalyst-test) (or [this](https://github.com/manjaro/packages-extra/blob/master/linux414-extramodules/catalyst/buildfix_kernel_4.14.patch)? wtf)

**Comment by imageguy** [comment]
Oh, apologies, you are right. I just checked the community forum, which is where A.I. lists new patches. I used to stumble on the problems anyway when I upgraded my own box, but my card died few weeks ago (after months of getting ever flakier) and my new card doesn't use fglrx. I'll cook a new patch tomorrow, but I can't test any more. I'm happy to keep rolling in the patches as they come out, but my lack of ability to test will be a problem.
